src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -7,5 +7,5 @@ public interface MachineProducer {
      * In your implementations - create 2-3 models, add them to the list and return
      * @return - the list of models
      */
-    List<Object> get();
+    List get();
```

Comment:
The `get()` method should also be parameterized. It should return a List of a specific Machine subtype, not a raw List. This ensures type safety and makes the code more readable.